static/bidder-info/gravite.yaml

```diff
@@ -0,0 +1,6 @@
+aliasOf: 'teqblaze'
+endpoint: "https://node1-rtb.gravite.net/pserver"
+gvlVendorID: 377
+maintainer:
+  email: "support@gravite.net"
```

Comment:
We sent an email to this address to confirm it is reachable. Please respond to the email with "received" to confirm.